modules/n1ql/examples/transactions/multiple.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/sh
+
+# tag::index[]
+curl http://localhost:8093/query/service \
+-u Administrator:password \
+-H 'Content-Type: application/json' \
+-d '{
+  "statement": "CREATE PRIMARY INDEX ON bookings;",
+  "query_context": "`travel-sample`.tenant_agent_00",
+}'
+# end::index[]
+
+# tag::transaction[]
+# tag::begin[]
+curl http://localhost:8093/query/service \
+-u Administrator:password \
+-H 'Content-Type: application/json' \
+-d '{
+  "statement": "BEGIN WORK",
+  "query_context": "`travel-sample`.tenant_agent_00",
+  "txtimeout": "2m",
+  "scan_consistency": "request_plus",
+  "durability_level": "none"
+}'
+# end::begin[]
+
+# tag::set[]
+curl http://localhost:8093/query/service \
+-u Administrator:password \
+-H 'Content-Type: application/json' \
+-d '{
+  "statement": "SET TRANSACTION ISOLATION LEVEL READ COMMITTED;",
+  "query_context": "`travel-sample`.tenant_agent_00",
+  "txid": "d81d9b4a-b758-4f98-b007-87ba262d3a51"
+}' # <1>
```

Comment:
the `<1>` text mentions the txid, so would be clearer on the line above.

Also unclear where this `txid` comes from? Is that in the return JSON for the `BEGIN WORK` call?

Reply:
Also I found the repeated `<1>` notes a bit odd - it makes sense when you notice they're the same thing, but looks like it could be an error until you check.

Reply:
I wasn't sure how the REST API would cope with a JSON payload containing a bash comment...  (for automated testing of examples)

Reply:
The same callout applies in each location though. I think it's fine to have multiple identical callout numbers in the code in this case. I used `<1>` rather than `<.>` to make sure that the callout number doesn't increment if I ever need to transclude the whole file.